<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QtTestClass</class>
 <widget class="QMainWindow" name="QtTestClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>757</width>
    <height>529</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QtTest</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0" colspan="2">
     <layout class="QHBoxLayout" name="horizontalLayout_7">
      <item>
       <widget class="QLabel" name="OriginImg">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="ProcessedImg">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_8">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLabel" name="lowerH">
            <property name="text">
             <string>lowerH</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSlider" name="lowerHSlider">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="lowerHSpinBox"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QLabel" name="upperH">
            <property name="text">
             <string>upperH</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSlider" name="upperHSlider">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="upperHSpinBox"/>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QLabel" name="lowerS">
            <property name="text">
             <string>lowerS</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSlider" name="lowerSSlider">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="lowerSSpinBox"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <widget class="QLabel" name="upperS">
            <property name="text">
             <string>upperS</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSlider" name="upperSSlider">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="upperSSpinBox"/>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_6">
          <item>
           <widget class="QLabel" name="lowerV">
            <property name="text">
             <string>lowerV</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSlider" name="lowerVSlider">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="lowerVSpinBox"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QLabel" name="upperV">
            <property name="text">
             <string>upperV</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSlider" name="upperVSlider">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="upperVSpinBox"/>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item row="1" column="1">
     <widget class="QPushButton" name="Start_PauseButton">
      <property name="text">
       <string>Start</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>757</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>Start_PauseButton</tabstop>
  <tabstop>upperHSlider</tabstop>
  <tabstop>upperHSpinBox</tabstop>
  <tabstop>lowerSSlider</tabstop>
  <tabstop>lowerSSpinBox</tabstop>
  <tabstop>upperSSlider</tabstop>
  <tabstop>upperSSpinBox</tabstop>
  <tabstop>lowerVSpinBox</tabstop>
  <tabstop>upperVSlider</tabstop>
  <tabstop>upperVSpinBox</tabstop>
 </tabstops>
 <resources>
  <include location="qttest.qrc"/>
 </resources>
 <connections/>
</ui>
